src/TabNav.tsx

```diff
@@ -28,8 +29,21 @@ const TabNavNav = styled.nav`
 export type TabNavProps = ComponentProps<typeof TabNavBase>
 
 function TabNav({children, 'aria-label': ariaLabel, ...rest}: TabNavProps) {
+  const customContainerRef = useRef<HTMLElement>(null)
+  const customStrategy = React.useCallback(() => {
+    if (customContainerRef.current) {
+      const tab = Array.from(customContainerRef.current.querySelectorAll<HTMLElement>('a.selected'))
```

Comment:
```suggestion
      const tabs = Array.from(customContainerRef.current.querySelectorAll<HTMLElement>('a[aria-selected="true"]'))
```
In case the styles change one day. Also because this is an array, prefer plural.